From f5906abf232081f4bc9ee40d512e388e403ca332 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Wed, 18 Mar 2020 00:43:58 +0100
Subject: [PATCH] KDSoapWSDiscoveryClientConfig.cmake.in: Add KDSoap to
 dependencies

Otherwise, revdeps (like kio-extras samba support) fail at cmake step:

CMake Error at smb/CMakeLists.txt:72 (add_library):
  Target "kio_smb" links to target "KDSoap::kdsoap" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?

CMake Error at smb/CMakeLists.txt:57 (add_library):
  Target "kio_smb_static" links to target "KDSoap::kdsoap" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
---
 KDSoapWSDiscoveryClientConfig.cmake.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/KDSoapWSDiscoveryClientConfig.cmake.in b/KDSoapWSDiscoveryClientConfig.cmake.in
index a2c572a..304cf64 100644
--- a/KDSoapWSDiscoveryClientConfig.cmake.in
+++ b/KDSoapWSDiscoveryClientConfig.cmake.in
@@ -5,5 +5,6 @@
 @PACKAGE_INIT@
 
 find_dependency(Qt5Core @REQUIRED_QT_VERSION@)
+find_dependency(KDSoap)
 
 include("${CMAKE_CURRENT_LIST_DIR}/KDSoapWSDiscoveryClientTargets.cmake")
-- 
2.26.2

